layouts/partials/nav.ejs

```diff
@@ -9,22 +9,13 @@
     <div class="attribution">
       <p>
         <% if (locals.byline) { %>
-          By <span class="author"><%= locals.byline %></span>.
+          Created by <span class="author"><%= byline %></span><%= bylineDateString(locals.createdAt) %>.
         <% } else if (locals.createdBy && !locals.byline) { %>
-          Created by <span class="author"><%= createdBy %></span> <%= createdAt %>.
+          Created by <span class="author"><%= createdBy %></span><%= bylineDateString(locals.createdAt) %>.
         <% } %>
         <% if (locals.lastUpdatedBy) { %>
-          <%# just encode the last updated date as data property to make relative time work for cache %>
-      Last edited by <span class="author"><%= lastUpdatedBy %></span> <span class="last-updated" data-last-updated="<%= modifiedAt %>"></span>.</p>
-          <!-- populate last updated time -->
-          <script>
-          (function () {
-            var $lastUpdated = $('.last-updated')
-            var utcUpdated = $lastUpdated.data('last-updated')
-            var updatedMoment = moment(utcUpdated)
-            $lastUpdated.html(updatedMoment.fromNow())
-          })()
```

Comment:
While it would be nice to move this logic server-side, timestamps do need to be calculated using moment on the client since our current cache holds onto the rendered page's HTML until the google doc is edited, which could lead to incorrect relative timestamps remaining in cache for long periods of time.

Reply:
Fixed!